operator/controllers/csiscaleoperator_controller.go

```diff
@@ -298,10 +279,15 @@ func (r *CSIScaleOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	if !cmExists || clustersStanzaModified {
 		err = ValidateCRParams(instance)
 		if err != nil {
-			logger.Error(fmt.Errorf("Failed to validate driver manifest parameters"), "")
+			message := "Failed to validate the Spectrum Scale cluster properties for the CSI driver to mount.\n" +
```

Comment:
"Failed to validate the Spectrum Scale cluster properties" should be sufficient

Reply:
Can this be changed to "Failed to validate Spectrum Scale CSI configurations"?